README.md

```diff
@@ -1,2 +1,9 @@
-# lc
-Load Control for beam
+lc
+=====
+
+load control app
```

Comment:
Could you add a brief description of the algorithm? For example, when backoff happens.